indico/cli/event.py

```diff
@@ -85,3 +87,39 @@ def import_(source_file, create_users, force, verbose, yes, category_id):
         sys.exit(1)
     db.session.commit()
     click.secho(event.external_url, fg='green', bold=True)
+
+
+@cli.command('create-series')
+@click.option('--no-title-sequence', is_flag=True,
+              help='Whether to show the series sequence in the event titles (lectures only)')
+@click.option('--no-links', is_flag=True,
+              help='Whether to show links to other events in the series on the event page')
+@click.argument('event_ids', nargs=-1, type=int)
+def create_series(no_title_sequence, no_links, event_ids):
+    """Create a series from a list of events."""
+    events = []
```

Comment:
```python
events = Event.query.filter(Event.id.in_(event_ids)~Event.is_deleted).all()
```

and for the missing events you can do something like `set(event_ids) - {e.id for e in events}`